sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md

```diff
@@ -363,6 +364,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AllowWebPropertyBagUpdateWhenDenyAddAndCustomizePagesIsEnabled
+Enables or disables web property bag update when DenyAddAndCustomizePages is enabled. When AllowWebPropertyBagUpdateWhenDenyAddAndCustomizePagesIsEnabled is set to $true, web property bag can be updated even if DenyAddAndCustomizePages is turned on when the user had AddAndCustomizePages (prior to DenyAddAndCustomizePages removing it).
```

Comment:
>AllowWebPropertyBagUpdateWhenDenyAddAndCustomizePagesIsEnabled [](http://example.com/codeflow?start=91&length=62)

use "this" to avoid repeating the param name

`When this is set to ...`